pkg/util/wait_test.go

```diff
@@ -0,0 +1,117 @@
+/*
+Copyright © 2019 Portworx
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+package util
+
+import (
+	"fmt"
+	"strings"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/assert"
+)
+
+const (
+	oneSecond  = 1 * time.Second
```

Comment:
This is great, but could you do it in Milliseconds? That way this test will not slow down the unit tests.

Reply:
Agree. Done with the changes to use milliseconds instead of seconds

Test run result:
Sivakumars-MacBook-Pro:util siva$ go test -v 
=== RUN   TestPxErrorMessage
--- PASS: TestPxErrorMessage (0.00s)
=== RUN   TestPxErrorMessagef
--- PASS: TestPxErrorMessagef (0.00s)
=== RUN   TestPrintPxErrorMessagef
--- PASS: TestPrintPxErrorMessagef (0.00s)
=== RUN   TestPxError
--- PASS: TestPxError (0.00s)
=== RUN   TestWaitFor
--- PASS: TestWaitFor (0.02s)
PASS
ok      github.com/portworx/px/pkg/util 0.046s